macos/FluentUITestApp/TestControlsViewController.swift

```diff
@@ -6,8 +6,8 @@
 import AppKit
 import FluentUITestViewControllers
 
-fileprivate struct Constants {
-	static let previousSelectionIndexUserDefaultsKey = "FluentUITestApp.previousSelectionIndexUserDefaultsKey"
+private struct Constants {
+	static let previousSelectionIndexUserDefaultsKey: String = "FluentUITestApp.previousSelectionIndexUserDefaultsKey"
```

Comment:
Do we have a rule that says we prefer explicit types? I thought we preffered them being inferred

Reply:
Yes, this is the rule:
**Explicit Type Interface Violation: Properties should have a type interface (explicit_type_interface)**

I think the inferred ones you're referring to are the variables:
**Redundant Type Annotation Violation: Variables should not have redundant type annotation (redundant_type_annotation)**

Reply:
I remember @markavitale preferred implicit to explicit when we started macOS to begin with, but obviously prefer following a common guide overall. Tagging anyway in case he has comments